runtimebp/cpu.go

```diff
@@ -86,14 +86,16 @@ func defaultMaxProcsFormula(n float64) int {
 // in bound of [min, max].
 //
 // Currently the default formula is NumCPU()*2-1 rounding up.
-func GOMAXPROCS(min, max int) int {
+func GOMAXPROCS(min, max int) (oldVal, newVal int) {
```

Comment:
🔕 

```suggestion
func GOMAXPROCS(min, max int) (old, new int) {
```

😅

Reply:
`new` is a keyword, otherwise I would not add `Val` prefix.

Reply:
Keywords, my old nemesis...